vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -21,12 +21,13 @@ In order to keep this vignette concise, we assume the following has been set up:
 2. All required R packages are installed, specifically.
 `r2dii.climate.stress.test`. Note that this may require installing additional
 packages. You will be prompted to install these in case any are missing.
-1. The user has obtained all relevant input files as described in the pdf manual.
-1. The user has successfully run at least the matching part of PACTA for Banks
+3. The user has successfully run at least the matching part of PACTA for Banks
 to produce the relevant project-specific raw input files.
-1. The input files are organized in a folder structure consistent with the one
+4. The user has obtained all relevant input files as described in the pdf manual.
+to produce the relevant project-specific raw input files.
+5. The input files are organized in a folder structure consistent with the one
 described in the vignette ["Set up project directories"](https://2degreesinvesting.github.io/r2dii.climate.stress.test/articles/articles/01-setup-project-directories.html).
```

Comment:
I am not sure. I fully understand when you used `"Title"` and when just `Title`.. maybe you can clarify for me?